sql/src/main/java/io/crate/planner/consumer/NestedLoopConsumer.java

```diff
@@ -230,8 +230,6 @@ public Plan visitTwoTableJoin(TwoTableJoin statement, ConsumerContext context) {
                 rightPlan = tmpPlan;
                 leftMerge = rightMerge;
                 rightMerge = null;
-                leftResultDesc = leftPlan.resultDescription();
```

Comment:
I know these are unused past this place but maybe we should still null them to avoid errors if we start using them (then they'd still be swapped which might lead to strange errors)